dist/OBS_debian/seamly2d.dsc

```diff
@@ -4,13 +4,13 @@ Binary: seamly2d
 Architecture: i386 amd64
 Version: 0.4.5
```

Comment:
Again here, we want to have ONE BUILD TO RULE THEM ALL, namely our Appimage build. Keeping up with differences in Linux versions, especially when there's such strife between Gnome and other desktop graphics pipelines, is not a good idea when we don't have a debian packager on our team.